helpers/controller.go

```diff
@@ -1,14 +1,19 @@
 package helpers
 
-import "github.com/beego/beego/v2/server/web"
+import (
+	"strings"
+
+	"github.com/beego/beego/v2/server/web"
+)
 
 var ActionsWithGetMethod = []string{"List", "New", "Edit", "Delete"}
 
 // SetControllerAttributes set attributes for controller
 func SetControllerAttributes(controller *web.Controller) {
 	controllerName, actionName := controller.GetControllerAndAction()
 
-	controller.Data["ControllerName"] = ToKebabCase(controllerName)
+	controllerName = strings.Replace(controllerName, "Controller", "", 1)
+	controller.Data["ControllerName"] = ToKebabCase( controllerName)
```

Comment:
Instead of adding the class `result-controller`, add the class `result` instead to align with the stylesheet filename